docs/hugging_face.md

```diff
@@ -0,0 +1,82 @@
+# Hugging Face 🤗
+
+## The Hugging Face Hub
+
+In addition to the official [pre-trained models](https://www.sbert.net/docs/pretrained_models.html), you can find over 500 `sentence-transformer` models on the [Hugging Face Hub](http://hf.co/models?library=sentence-transformers&sort=downloads) covering a range of tasks. You can filter to find particular models, such as searching for [`feature-extraction`](https://huggingface.co/models?library=sentence-transformers&pipeline_tag=feature-extraction&sort=downloads) for models for generating embeddings, or [`sentence-similarity`](https://huggingface.co/models?library=sentence-transformers&pipeline_tag=sentence-similarity&sort=downloads) for sentence similarity models.
```

Comment:
Maybe let's clarify the wording around feature-extraction and sentence-similarity as the models can really do both tasks.

Reply:
Good point. I'm not quite sure I understand the feature-extraction and sentence-similarity tags in these models, to be honest. Do they just affect the widget on the model page? I'm confused because even though I don't see a [Sentence Similarity pipeline](https://huggingface.co/docs/transformers/v4.20.1/en/main_classes/pipelines), or a [Sentence Similarity task for the Inference API](https://huggingface.co/docs/api-inference/detailed_parameters), I'm able to call the inference endpoint for `sentence-transformers/all-mpnet-base-v2` at both `feature-extraction` and `sentence-similarity`.

If the tags are only being used to define the widget, I can remove the content here about filtering.

Reply:
We should add sentence similarity to https://huggingface.co/docs/api-inference/detailed_parameters I think :)

Reply:
Also https://huggingface.co/docs/transformers/v4.20.1/en/main_classes/pipelines is transformers, not the community API. See https://www.notion.so/huggingface2/Pipelines-3a38a9c3cdea4b60af0db88ee789d70c

Reply:
Opened an issue for `api-inference` btw, will do this next week: https://github.com/huggingface/api-inference/issues/1069